src/pages/billing/Project.js

```diff
@@ -60,34 +60,35 @@ const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ needs
   ])
 })
 
-const rowBase = { display: 'flex', alignItems: 'center', width: '100%' }
-
-const ExpandedInfoRow = Utils.memoWithName('ExpandedInfoRow', ({ title, details, additionalInfo }) => {
+const ExpandedInfoRow = Utils.memoWithName('ExpandedInfoRow', ({ title, details, errorMessage }) => {
   const expandedInfoStyles = {
-    row: { ...rowBase, marginTop: '0.5rem', height: '1rem' },
-    title: { fontWeight: 600, width: '20%', paddingRight: '1rem', height: '1rem' },
-    details: { flexGrow: 1, width: '20%', paddingRight: '1rem', height: '1rem', ...Style.noWrapEllipsis },
-    additionalInfo: { flexGrow: 1 }
+    row: { display: 'flex', justifyContent: 'flex-start', alignItems: 'flex-start' },
+    title: { fontWeight: 600, width: '20%', padding: '0.5rem 1rem 0 2rem', height: '1rem' },
+    details: { width: '20%', marginTop: '0.5rem', height: '1rem', ...Style.noWrapEllipsis },
+    errorMessage: {
+      padding: '0.5rem', width: '55%', backgroundColor: colors.light(0.3),
+      border: `solid 2px ${colors.danger(0.3)}`, borderRadius: '5px'
```

Comment:
React allows this shorthand for px values if you like:
```suggestion
      border: `solid 2px ${colors.danger(0.3)}`, borderRadius: 5
```